polkadot/node/core/approval-voting/src/import.rs

```diff
@@ -217,7 +220,17 @@ async fn imported_block_info<Context>(
 	let session_info = get_session_info(env.runtime_info, ctx.sender(), block_hash, session_index)
 		.await
 		.ok_or(ImportedBlockInfoError::SessionInfoUnavailable)?;
-
+	let enable_v2_assignments = request_node_features(block_hash, session_index, ctx.sender())
```

Comment:
I think it makes sense to add node features in `ExtendedSessionInfo` and call `get_extended_session_info` above.